### PR TITLE
refactor[test]: Segregated the backup and restore test based on the storage engine

### LINKED ISSUE #546: Refactor the backup and restore test case to segregate the tests into individual play books
- In e2e backup and restore we have multiple test cases coupled into single test case segregate those into simpler test cases.
- And change the end to end backup and restore test to modify as like backup on one stage and restore into one stage.

### PR DESCRIPTION
**Pull request by nsathyaseelan**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Checklist**

* [ ] Does this PR have a corresponding GitHub issue?
* [ ] Have you included relevant README for the chaoslib/experiment with details?
* [ ] Have you added debug messages where necessary? 
* [ ] Have you added task comments where necessary? 
* [ ] Have you tested the changes for possible failure conditions?
* [ ] Have you provided the positive & negative test logs for the litmusbook execution?
* [ ] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [ ] Have you used non-shell/command modules for Kubernetes tasks?
* [ ] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any? 
* [ ] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [ ] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [ ] Do the artifacts follow the appropriate directory structure? 
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks? 
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?  
* [ ] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [ ] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [ ] Can you suggest the minimal resource requirements for the litmusbook execution?
* [ ] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [ ] Has the litmusbooks been linted? 

**Special notes for your reviewer**:

### CODE REVIEW

File: experiments/functional/backup_and_restore/cstor/run_litmus_test.yml [hunk 1]
```diff
@@ -76,13 +79,13 @@ spec:
...
           # supported values are: true & false (Deprovision velero namespace after test completion)
           - name: DEPROVISION_VELERO
             value: "false"
+
+          # supported values are: BACKUP & RESTORE
+          - name: VELERO_ACTION
+            value: "BACKUP"            
              
         command: ["/bin/bash"]
         args: ["-c", "ansible-playbook ./experiments/functional/backup_and_restore/test.yml -i /etc/ansible/hosts -vv; exit 0"]
```
Comment (w3aman) [inline experiments/functional/backup_and_restore/cstor/run_litmus_test.yml:91]: <file name> ==> backup_and_restore/cstor/test.yml

Comment (w3aman) [inline experiments/functional/backup_and_restore/cstor/run_litmus_test.yml]: file name===> backup_and_restore/cstor/test.yml

File: experiments/functional/backup_and_restore/localpv/run_litmus_test.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+          # supported values are: true & false (Deprovision velero namespace after test completion)
+          - name: DEPROVISION_VELERO
+            value: "false"
+
+          # supported values are: backup & restore
+          - name: VELERO_ACTION
+            value: "BACKUP"            
+             
+        command: ["/bin/bash"]
+        args: ["-c", "ansible-playbook ./experiments/functional/backup_and_restore/test.yml -i /etc/ansible/hosts -vv; exit 0"]
```
Comment (w3aman) [inline experiments/functional/backup_and_restore/localpv/run_litmus_test.yml]: here also ===> file name ?

File: experiments/functional/backup_and_restore/jiva/run_litmus_test.yml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+          # supported values are: true & false (Deprovision velero namespace after test completion)
+          - name: DEPROVISION_VELERO
+            value: "false"
+
+          # supported values are: backup & restore
+          - name: VELERO_ACTION
+            value: "BACKUP"            
+             
+        command: ["/bin/bash"]
+        args: ["-c", "ansible-playbook ./experiments/functional/backup_and_restore/test.yml -i /etc/ansible/hosts -vv; exit 0"]
```
Comment (w3aman) [inline experiments/functional/backup_and_restore/jiva/run_litmus_test.yml]: path ==> /jiva/test.yml

**Review by w3aman — APPROVED** [review]
/looks good

**Review by gprasath — APPROVED** [review]

**Pull request merged**